packages/web-app-files/src/views/Favorites.vue

```diff
@@ -117,6 +118,22 @@ export default defineComponent({
       ViewModeConstants.tilesView
     ])
 
+    const loadResourcesEventToken = ref(null)
+
+    onMounted(() => {
+      loadResourcesEventToken.value = eventBus.subscribe(
+        'app.files.list.removeFromFavorites',
+        (resourceId: string) => {
+          store.commit('Files/REMOVE_FILES', [{ id: resourceId }])
```

Comment:
@elizavetaRa @kulmann I opted for a simple store update over the full loadResourcesTask request in the first commit in this PR since we already know that the resource in question has been updated in the server (the event gets published in the `.then()` part in the (un)favorite-action)